### ISSUE TITLE
[Question]: 

### ISSUE THREAD

**Issue by SudharakaA** [dockur/windows#867]
### Is your question not already answered in the FAQ?

- [X] I made sure the question is not listed in the [FAQ](https://github.com/dockur/windows/blob/master/readme.md).

### Is this a general question and not a technical issue?

- [X] I am sure my question is not about a technical issue.

### Question

Is this compatible with M1 Macs?

**Comment by JacksonBurns** [comment]
Yes, with some limitations: https://github.com/dockur/windows/issues/317
